components/elm/src/main/ncdio_pio.F90.in

```diff
@@ -1513,11 +1513,9 @@ end subroutine ncd_io_{DIMS}d_{TYPE}_glob
                 !find first non-zero occurrence in the "start" array
                 n = 3
                 do while(start(n) == 0)
+                   n_time_ind = n ! time index for the start array
```

Comment:
In the latest code, n_time_ind won't be initialized if start(3) !=0

Reply:
This has also been fixed.